libs/SalesforceSDKCore/SalesforceSDKCore/Classes/UserAccount/SFUserAccountManager.m

```diff
@@ -319,15 +325,81 @@ - (void) setLoginViewControllerConfig:(SFSDKLoginViewControllerConfig *)config {
 
 #pragma  mark - login & logout
 
-- (BOOL)handleIDPAuthenticationResponse:(NSURL *)appUrlResponse options:(nonnull NSDictionary *)options {
-    [SFSDKCoreLogger d:[self class] format:@"handleIDPAuthenticationResponse %@",[appUrlResponse description]];
+- (BOOL)handleIDPAuthenticationResponse:(NSURL *)appUrlResponse options:(NSDictionary *)options {
```

Comment:
I think Android and iOS did not name the various requests / responses which is okay, may be some comments might help.

For reference, on Android we have:
- IDPLoginRequest: sent by IDP to SP for IDP initiated login flow (contains a user hint)
- IDPLoginResponse: sent by SP to IDP at the end of the flow (either right after the IDPLoginRequest if the user hinted was available, or after getting the code and logging in) => probably no equivalent on iOS 
- SPLoginRequest: sent by SP to IDP for SP initiated login flow or during IDP initiated login flow if SP does not have the hinted user (contains code challenge)
- SPLoginResponse: sent by IDP to IDP in response to SPLoginRequest (contains code)

Reply:
thanks!

Reply:
I updated existing command names to line up more with Android, and then added `SFSDKIDPAuthCodeLoginRequestCommand` => sent by IDP to SP for IDP initiated login flow using the keychain to store the code verifier